src/app/categories/components/categoryCard/categoryCard.tsx

```diff
@@ -49,7 +49,7 @@ export const CategoryCard = (props: CategoryCardProps) => {
 
       <Flex gap={'1rem'} align={'center'}>
         <Button className={classes.button}>
-          <Link href={`/list`}>
+          <Link href={`/subcategory`}>
```

Comment:
Will not agry here with the change because URL is mostly for user. And for user this instance is just list.

Reply:
but the folder name must be the same as the url name in nextjs, isn't it?

Reply:
yes sure, my bad, sorry 😆